src/main/host/syscall_handler.c

```diff
@@ -50,6 +50,12 @@
 #define SYS_copy_file_range 326
 #endif
 
+static bool _useMM = true;
+OPTION_EXPERIMENTAL_ENTRY("no-memory-manager", 0, G_OPTION_FLAG_REVERSE, G_OPTION_ARG_NONE, &_useMM,
```

Comment:
FYI, I think Tor's options use the `--enable-XXX` and `--disable-XXX` format to enable and disable boolean features. Probably not worth changing here, since I don't think we use this consistently for the other options anyway.

Reply:
Done.

I think one reason I previously went with integer types here is so that in case the default changes we don't need to change the flag. I suppose we could always just define both the `enable-X` and `disable-X` flags though.